pkg/nfs/controllerserver.go

```diff
@@ -330,26 +467,26 @@ func (cs *ControllerServer) copyFromVolume(ctx context.Context, req *csi.CreateV
 		volCap = req.GetVolumeCapabilities()[0]
 	}
 	if err = cs.internalMount(ctx, srcVol, nil, volCap); err != nil {
-		return status.Errorf(codes.Internal, "failed to mount src nfs server: %v", err.Error())
+		return status.Errorf(codes.Internal, "failed to mount src nfs server: %v", err)
```

Comment:
mini cleanup after https://github.com/kubernetes-csi/csi-driver-nfs/pull/426 - calling `err.Error()` was not necessary